platform/i18n/src/locales/en-US/index.js

```diff
@@ -1,21 +1,21 @@
-import AboutModal from "./AboutModal.json";
-import Buttons from "./Buttons.json";
-import CineDialog from "./CineDialog.json";
-import Common from "./Common.json";
-import Header from "./Header.json";
-import MeasurementTable from "./MeasurementTable.json";
-import StudyList from "./StudyList.json";
-import UserPreferencesModal from "./UserPreferencesModal.json";
+import AboutContent from './AboutContent.json';
```

Comment:
Does updating translations like this work? I think they may get blown out on re-sync? Rodrigo had to do the following in the past:

https://github.com/OHIF/Viewers/pull/998#issuecomment-538454799

Reply:
Done! let's keep the namespace as Modal for now.